checkstyle-tester/diff.groovy

```diff
@@ -103,19 +109,20 @@ def areValidCliOptions(cliOptions) {
     return valid
 }
 
-def isValidCheckstyleConfigsCombination(config, baseConfig, patchConfig) {
+def isValidCheckstyleConfigsCombination(config, baseConfig, patchConfig, patchOnly) {
     def valid = true
-    if (config && patchConfig
+    if (baseConfig && !patchConfig
+            || config && patchConfig
             || config && baseConfig
             || config && patchConfig && baseConfig) {
-        err.println "Error: you should specify either \'config\' or \'baseConfig\' and \'patchConfig\'!"
+        err.println "Error: you should specify either \'config\' or \'baseConfig\' and \'patchConfig\', or \'patchConfig\' only!"
```

Comment:
I think there should be a comma before first 'or' too to more clearly make the separation from the 3 items.